README.md

```diff
@@ -1 +1,92 @@
 # scrooge-v2
+## Endpoint
+### POST /events/_create
+request
+```
+{
+    "name": "Koiki Camp",    ... (required)
+    "transferCurrency": "JPY"        ... (required)
+}
+```
+
+response (201 status)
+```
+{
+    "name": "Koiki Camp",
+    "id": "5a226c2d7c245e14f33fc5a8",
+    "createdAt": "2017-12-02T16:52:45.52",
+    "updatedAt": "2017-12-02T16:52:45.52",
+    "transferCurrency": "JPY",
+    "groups": [{ "scrooges": [], "members": []}],
```

Comment:
We need to have group name in each group.
```
"groups": [{ "name": "...", "scrooges": [], "members": []}],
```

Reply:
updated